benchmark/buffers/buffer-copy.js

```diff
@@ -2,7 +2,7 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  bytes: [0, 8, 128, 32 * 1024],
+  bytes: [8, 128, 1024],
```

Comment:
Do we expect it to be always a power of two?

Reply:
No, jumping from 128 to 1024 should be a good branch of measurement